### ISSUE TITLE
Duplicated celery setting for django?

### ISSUE THREAD

**Issue by nemesisdesign** [getsentry/raven-python#685]
Here: https://docs.getsentry.com/hosted/clients/python/integrations/django/#additional-settings

it says:

*If you are also using Celery, there is a handler being automatically registered for you that captures the errors from workers. The default logging level for that handler is logging.ERROR and can be customized using this setting:*

```python
SENTRY_CELERY_LOGLEVEL = logging.INFO
RAVEN_CONFIG = {
    'CELERY_LOGLEVEL': logging.INFO
}
```

Is it intentional? Does the setting need to be duplicated as shown?

PS: tried to dig into the code but could not figure it out

**Comment by dcramer** [comment]
We should probably kill the first one. Technically we support both, but theres no reason that anyone should care about that.

**Comment by nemesisdesign** [comment]
Sounds reasonable to remove the first one from the docs.

**Comment by nemesisdesign** [comment]
Curious thing, I cannot find that bit of text here: https://github.com/getsentry/raven-python/blob/master/docs/integrations/django.rst

Am I looking at the wrong file?

**Comment by mitsuhiko** [comment]
It's that file, it's in the source. Github just fails at rendering it.

**Comment by mitsuhiko** [comment]
Fixed in bbc7e1523c3723c5fe479494e03abb0e0400ecf1

**Comment by nemesisdesign** [comment]
Ok sorry I should have looked more in deep, I wanted to send a small patch but @mitsuhiko was faster ;-)
Thx
